src/Onboarding/EulaModal.tsx

```diff
@@ -2,249 +2,119 @@ import React, { useEffect, useState, FunctionComponent } from "react"
 import {
   TouchableOpacity,
   Linking,
-  Modal,
   StyleSheet,
   View,
   SafeAreaView,
-  StatusBar,
-  Image,
   ActivityIndicator,
 } from "react-native"
 import { useTranslation } from "react-i18next"
 import loadLocalResource from "react-native-local-resource"
 import WebView, { WebViewNavigation } from "react-native-webview"
 import { SvgXml } from "react-native-svg"
+import { useNavigation } from "@react-navigation/native"
 
-import { Button, RTLEnabledText } from "../components"
-import en from "../locales/eula/en.html"
-import es_PR from "../locales/eula/es_PR.html"
-import ht from "../locales/eula/ht.html"
+import enEulaHtml from "../locales/eula/en.html"
+import esPREulaHtml from "../locales/eula/es_PR.html"
+import htEulaHtml from "../locales/eula/ht.html"
 
-import { Icons, Images } from "../assets"
-import { Spacing, Buttons, Colors, Typography, Forms } from "../styles"
+import { Icons } from "../assets"
+import { Layout, Colors, Iconography, Spacing, Outlines } from "../styles"
+import { useStatusBarEffect } from "../navigation"
 
-type CloseModalIconProps = {
-  closeModal: () => void
-  label: string
-}
-
-const CloseModalIcon: FunctionComponent<CloseModalIconProps> = ({
-  closeModal,
-  label,
-}) => {
-  const size = 20
-  return (
-    <TouchableOpacity
-      accessibilityLabel={label}
-      accessible
-      style={style.closeIcon}
-      onPress={closeModal}
-    >
-      <SvgXml
-        color={Colors.icon}
-        xml={Icons.Close}
-        width={size}
-        height={size}
-      />
-    </TouchableOpacity>
-  )
-}
-
-interface CheckboxProps {
-  label: string
-  onPress: () => void
-  checked?: boolean
-}
-
-const Checkbox: FunctionComponent<CheckboxProps> = ({
-  label,
-  onPress,
-  checked,
-}) => {
+const LoadingIndicator = () => {
   return (
-    <TouchableOpacity
-      style={style.checkbox}
-      onPress={onPress}
-      accessible
-      accessibilityRole="checkbox"
-      accessibilityLabel={label}
-    >
-      <Image
-        source={checked ? Images.BoxCheckedIcon : Images.BoxUncheckedIcon}
-        style={style.checkboxIcon}
-      />
-      <RTLEnabledText style={style.checkboxText}>{label}</RTLEnabledText>
-    </TouchableOpacity>
+    <View style={style.loadingIndicator}>
+      <ActivityIndicator size={"large"} color={Colors.darkGray} />
+    </View>
   )
 }
 
 const DEFAULT_EULA_URL = "about:blank"
-
 type AvailableLocale = "en" | "es_PR" | "ht"
```

Comment:
should we create a ticket somewhere to start to work on the greek translation?

Reply:
Yeah - we should let whoever needs to know to get the Greek EULA html set up.